### PR TITLE
[NFC] Add ImportSet::dump() + fix module dumpRef

### PR DESCRIPTION
**Pull request by beccadax**
Some tiny debug utility improvements I'm pulling out of #34556, which is only tangentially related.

### CODE REVIEW

**Comment by beccadax** [comment]
@swift-ci smoke test

File: lib/AST/ImportCache.cpp [hunk 1]
```diff
@@ -63,6 +63,24 @@ void ImportSet::Profile(
...
+
+  llvm::errs() << "TopLevelImports:";
+  for (auto import : getTopLevelImports()) {
+    llvm::errs() << "\n- ";
+    simple_display(llvm::errs(), import);
+  }
+  llvm::errs() << "\n";
+
+  llvm::errs() << "TransitiveImports:";
+  for (auto import : getTopLevelImports()) {
```
Comment (beccadax) [inline lib/AST/ImportCache.cpp:77]: ಠ_ಠ

**Comment by beccadax** [comment]
@swift-ci please smoke test

**Review by varungandhi-apple — APPROVED** [review]
Thanks! It's a nice little QoL improvement.

**Comment by beccadax** [comment]
> clang-10: error: unable to execute command: Killed: 9

@swift-ci please smoke test macOS platform

**Pull request merged**